crt_portal/cts_forms/templates/landing.html

```diff
@@ -194,7 +194,7 @@ <h2>{% trans "Common civil rights violations" %}</h2>
                           data-example{{ forloop.counter0 }}="{{ example }}"
                         {% endfor %}
                       >
-                        {{ value.description }}
+                        <a href="#">{{ value.description }}</a>
```

Comment:
When running locally, click this links no longer has an effect. The examples below the input aren't upating.

Reply:
Right you are. Sigh. Fixing.